homework/Contributor.js

```diff
@@ -14,6 +14,22 @@ class Contributor {
    */
   render(container) {
     // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
+    const list2 = Util.createAndAppend('li', container);
+    Util.createAndAppend('a', list2, {
```

Comment:
I don't think the way you are adding HTML elements here is the best way to do. Because if you inspect the app on browser, the anchor(`a`) tag is added as a sibling to the paragraph element(`p`) and has got no height or width of its own. And hence it is not clickable from mouse. You can only get to the anchor element through keyboard. Could you please check this once? 🙂

Reply:
@yash-kapila  thank you for commenting on my PR. 
i have made the requested changes.